openupgradelib/openupgrade_merge_records.py

```diff
@@ -54,6 +58,9 @@ def _change_foreign_key_refs(env, model_name, record_ids, target_record_id,
             env.cr.execute('ROLLBACK TO SAVEPOINT sp1')
             if error.pgcode != UNIQUE_VIOLATION:
                 raise
+            elif error.pgcode == UNDEFINED_COLUMN:
+                # extra_where is introducing a bad column. Ignore this table thus
```

Comment:
Dont you think you should log a warning here ? It means that thé call to thé function was incorrect. Or did i missed something ?

Reply:
It was incorrect, but with that exact code, so the only possible way to do that is through the extra_where (the rest of columns are got from the FK references, which are valid). I don't think we should hammer logs with that, and more on the use case we want to use it.

Reply:
This line is too long, that's why CI is red.
![Selection_203](https://user-images.githubusercontent.com/25005517/135333358-5c288980-fd1b-4520-9cf4-c92867334c06.png)